conf/app.routes

```diff
@@ -14,13 +14,16 @@ GET         /                                                          controlle
 GET         /applications                                              controllers.IndexController.getApplications(page: Int ?= 1)
 GET         /rulings                                                   controllers.IndexController.getRulings(page: Int ?= 1)
 GET         /applications-and-rulings                                  controllers.IndexController.getApplicationsAndRulings(page: Int ?= 1, sortBy: Option[SortField], order: Option[SortDirection])
+GET         /bta                                                       controllers.BTARedirectController.bta
+GET         /btaRedirect                                               controllers.BTARedirectController.btaRedirect
```

Comment:
can we keep this in lower case, or like `bta-redirect` just for consistency

Reply:
done